README.md

```diff
@@ -544,6 +550,24 @@ import { setRenderPartialOnly } from '@americanexpress/one-app-ducks';
 dispatch(setRenderPartialOnly(true));
 ```
 
+##### `setRenderTextOnly`
+
+Use this action creator to render text only from a holocron module, rather than a HTML.
+
+| Argument | Type | Description |
+|---|---|---|
+| `setTextOnly` | `Boolean` | (required) set whether to return text instead of HTML |
```

Comment:
if we say that `setTextOnly` accepts a boolean does that mean if set to false it would return the html? Should we then set this to a default value of `true` ?

Reply:
If you set it to true it will remove the html. It has to be false by default (no change)

Reply:
Thank you :-)

Reply:
Yeah I like that structure better, updated here https://github.com/americanexpress/one-app-ducks/pull/17/commits/cc41331600b59176f2c61fd3f095565036f0d25b also updated the default state to reflect the new options parameter, @10xLaCroixDrinker let me know what you think